configure.ac

```diff
@@ -7,6 +7,7 @@ AM_SILENT_RULES([yes])
 AC_PREFIX_DEFAULT(/usr)
 
 AC_CANONICAL_HOST
+AX_PROG_CC_FOR_BUILD
```

Comment:
Do you not need AX_PROG_CXX_FOR_BUILD for ptimec.cc ?

Reply:
Seemed like a C program by the looks of it and compiles fine with CC. I can change it to CXX.